src/vm/gccover.cpp

```diff
@@ -940,6 +943,41 @@ bool replaceInterruptibleRangesWithGcStressInstr (UINT32 startOffset, UINT32 sto
 }
 #endif
 
+#if defined(_TARGET_ARM64_)
+// Is this a call instruction to JIT_RareDisableHelper()
+// We do not need to insert GCStress instruction at this call
+// For arm64 call to jithelpers happens via a stub.
+// For other architectures call does not happen via stub.
+// For other architecures we can get the target directly by calling getTargetOfCall().
+// This is not the case for arm64 so need to decode the stub
+// instruction to find the actual jithelper target. 
+// For other architecture we detect call to JIT_RareDisableHelper 
+// in function OnGcCoverageInterrupt() since getTargetOfCall() can
+// get the actual jithelper target.
+bool isCallToStopForGCJitHelper(SLOT instrPtr)
+{
+   if (((*reinterpret_cast<DWORD*>(instrPtr)) & 0xFC000000) == 0x94000000)
+   {
+       // call through immediate
+       int imm26 = ((*((DWORD*)instrPtr)) & 0x03FFFFFF)<<2;
+       // SignExtend the immediate value.
+       imm26 = (imm26 << 4) >> 4;
+       DWORD* target = (DWORD*) (instrPtr + imm26);
+       // Call to jithelpers happens via jumpstub
```

Comment:
Baking in this fact seems likely to be fragile, but it probably OK if this is for GC Stress only